demo/src/main/java/com/orange/ods/demo/ui/components/ComponentDetailScreen.kt

```diff
@@ -36,74 +38,65 @@ import com.orange.ods.demo.ui.components.utilities.ComponentHeader
+                            }
+                            Box(modifier = Modifier.height(dimensionResource(id = R.dimen.list_single_line_item_height)), contentAlignment = Alignment.Center) {
+                                OdsTextSubtitle2(
+                                    modifier = Modifier
+                                        .padding(horizontal = dimensionResource(id = R.dimen.ods_screen_horizontal_margin)),
+                                    text = stringResource(id = section.titleRes)
+                                )
+                            }
+
+                        }
+                        variants.forEach { variant ->
+                            ComponentDetailLinkItem(stringResource(id = variant.titleRes)) { onVariantClick(variant.id) }
+                        }
+                    }
+                }
             }
         }
     }
 }
 
 @ExperimentalMaterialApi
 @Composable
-fun ComponentDetailWithVariants(component: Component, onVariantClick: (Long) -> Unit) {
-    Column(
-        modifier = Modifier
-            .verticalScroll(rememberScrollState())
-            .padding(bottom = dimensionResource(id = R.dimen.ods_screen_vertical_margin))
-    ) {
-        ComponentHeader(
-            imageRes = component.imageRes,
-            imageAlignment = component.imageAlignment,
-            description = component.descriptionRes
-        )
-        Column(
-            modifier = Modifier.padding(top = dimensionResource(id = R.dimen.spacing_m))
-        ) {
-            component.variants.groupBy { it.section }.onEachIndexed { index, (section, variants) ->
-                section?.let {
-                    if (index > 0) {
-                        Divider(modifier = Modifier.padding(top = dimensionResource(id = R.dimen.spacing_s)))
-                    }
-                    Box(modifier = Modifier.height(dimensionResource(id = R.dimen.list_single_line_item_height)), contentAlignment = Alignment.Center) {
-                        OdsTextSubtitle2(
-                            modifier = Modifier
-                                .padding(horizontal = dimensionResource(id = R.dimen.ods_screen_horizontal_margin)),
-                            text = stringResource(id = section.titleRes)
-                        )
-                    }
-
-                }
-                variants.forEach { variant ->
-                    OdsListItem(text = stringResource(id = variant.titleRes), modifier = Modifier.clickable {
-                        onVariantClick(variant.id)
-                    })
-                }
-
-            }
-        }
-    }
+private fun ComponentDetailLinkItem(label: String, onClick: () -> Unit) {
+    OdsListItem(
+        icon = { Icon(painter = painterResource(id = R.drawable.ic_play_outline), contentDescription = null) },
+        text = label,
+        modifier = Modifier.clickable { onClick() })
```

Comment:
This does not change anything but we should be using `iconType(OdsListItemIconType.Icon)` and `OdsListItemIcon` instead of `Icon` in order to benefit from future updates in `OdsLitItem`.